src/omv/ports/nrf/sensor.c

```diff
@@ -812,6 +820,29 @@ bool sensor_get_auto_rotation()
     return sensor.auto_rotation;
 }
 
+int sensor_set_framebuffers(int count)
+{
+    if (framebuffer_get_vbuffer_count() == count) {
+        return 0;
+    }
+
+    if (count < 1) {
+        return -1;
+    }
+
+    // This can fail if the user passes a ridiculous count.
+    if (!framebuffer_set_vbuffer_count(count)) {
+        return -1;
+    }
+
+    return 0;
+}
```

Comment:
```suggestion
    // This can fail if the user passes a ridiculous count.
    return framebuffer_set_buffers(count);
}
```
It's best to move these sanity checks to framebuffer_set_buffers, to avoid duplicating them in every sensor.c.

Reply:
I can't. DCMI abort is in the STM32 one.

Reply:
Moved.